blueprint/core/blueprint_ShadowView.cpp

```diff
@@ -83,6 +83,111 @@ namespace blueprint
+
+            // Flex dimensions
+            juce::Identifier flex            = "flex";
+            juce::Identifier flexGrow        = "flex-grow";
+            juce::Identifier flexShrink      = "flex-shrink";
+            juce::Identifier flexBasis       = "flex-basis";
+            juce::Identifier width           = "width";
+            juce::Identifier height          = "height";
+            juce::Identifier minWidth        = "min-width";
+            juce::Identifier minHeight       = "min-height";
+            juce::Identifier maxWidth        = "max-width";
+            juce::Identifier maxHeight       = "max-height";
+            juce::Identifier aspectRatio     = "aspect-ratio";
+
+            // Margin
+            juce::Identifier marginMetaProp   = "margin";
+            juce::Identifier marginLeft       = juce::String("margin-") + YGEdgeToString(YGEdgeLeft);
+            juce::Identifier marginRight      = juce::String("margin-") + YGEdgeToString(YGEdgeRight);
+            juce::Identifier marginTop        = juce::String("margin-") + YGEdgeToString(YGEdgeTop);
+            juce::Identifier marginBottom     = juce::String("margin-") + YGEdgeToString(YGEdgeBottom);
+            juce::Identifier marginStart      = juce::String("margin-") + YGEdgeToString(YGEdgeStart);
+            juce::Identifier marginEnd        = juce::String("margin-") + YGEdgeToString(YGEdgeEnd);
+            juce::Identifier marginHorizontal = juce::String("margin-") + YGEdgeToString(YGEdgeHorizontal);
+            juce::Identifier marginVertical   = juce::String("margin-") + YGEdgeToString(YGEdgeVertical);
+
+            // Padding
+            juce::Identifier paddingMetaProp   = "padding";
+            juce::Identifier paddingLeft       = juce::String("padding-") + YGEdgeToString(YGEdgeLeft);
+            juce::Identifier paddingRight      = juce::String("padding-") + YGEdgeToString(YGEdgeRight);
+            juce::Identifier paddingTop        = juce::String("padding-") + YGEdgeToString(YGEdgeTop);
+            juce::Identifier paddingBottom     = juce::String("padding-") + YGEdgeToString(YGEdgeBottom);
+            juce::Identifier paddingStart      = juce::String("padding-") + YGEdgeToString(YGEdgeStart);
+            juce::Identifier paddingEnd        = juce::String("padding-") + YGEdgeToString(YGEdgeEnd);
+            juce::Identifier paddingHorizontal = juce::String("padding-") + YGEdgeToString(YGEdgeHorizontal);
+            juce::Identifier paddingVertical   = juce::String("padding-") + YGEdgeToString(YGEdgeVertical);
+
+            //Position
+            juce::Identifier positionLeft      = YGEdgeToString(YGEdgeLeft);
+            juce::Identifier positionRight     = YGEdgeToString(YGEdgeRight);
+            juce::Identifier positionTop       = YGEdgeToString(YGEdgeTop);
+            juce::Identifier positionBottom    = YGEdgeToString(YGEdgeBottom);
+            //TODO: Any need for other edge values? i.e. start, end ?
```

Comment:
I don't think the other values from `ValidEdges` are actually needed here?

Reply:
Agreed, and I think we can remove the `start` and `end` props from padding and margin as well tbh. They're properties that aren't supported in standard CSS so it seems sensible to ignore them here